system/datatypes/settings.php

```diff
@@ -304,16 +306,29 @@ function editTheme() {
 			if (isAdmin()) {
 				ob_start();
 				echo '<select name="editTheme" id="editTheme">';
-				foreach (scandir('data/themes/') as $theme) {
-					if ('.' !== $theme[0]) {
-						echo '<option' . (Hypha::$data->theme === $theme ? ' selected' : '') . '>' . htmlspecialchars($theme) . '</option>';
-					}
+				$themes = $this->getThemes();
+				foreach ($themes as $theme) {
+					echo '<option' . (Hypha::$data->theme === $theme ? ' selected' : '') . '>' . htmlspecialchars($theme) . '</option>';
```

Comment:
This is a good change, but looks like it should be in a separate commit.

Reply:
I introduced the function in this commit, might as well use it here.